test/testcafe/framework/page-objects/BasePageObject.js

```diff
@@ -72,4 +72,13 @@ export default class BasePageObject {
   async clickGoBackLink() {
```

Comment:
can this be removed?

Reply:
I left all the code corresponding to `Go back` as is for now, we may need it for https://github.com/okta/okta-signin-widget/pull/1316#discussion_r456182878 on a second thought.

Reply:
note: may need for https://oktainc.atlassian.net/browse/OKTA-314043